youtube-videos-search/.env

```diff
@@ -0,0 +1,2 @@
+//API_KEY = AIzaSyAKViahE5uxk46ppvgxNM6NKIVDHL8_KTo
+API_KEY = AIzaSyCLyrdmpOMSi7SVR9A2WgzmidyszWH20oo
```

Comment:
be careful in next time with API keys and other secure information 
as it shouldn't go into repo